recipes/python-mumps/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "python-mumps" %}
+{% set version = "0.0.1a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python_mumps-{{ version }}.tar.gz
+  sha256: 56e2d9b3c0d7ed1c785e1c544617ed76d7ee250fc8d46e098928dbf531819881
+
+build:
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - python
+    - cython
+    - numpy
+    - {{ compiler('c') }}
+  host:
+    - python
+    - cython >=3.0.2
+    - meson >=1.1.0
+    - meson-python >=0.15.0
+    - numpy >=1.24
```

This should be pinned to the oldest possible.
```suggestion
    - numpy 1.24
```

---

Resolved, and also downgraded to 1.23 to be compatible with NEP 29.